draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -1404,8 +1409,9 @@ Transfer-Encoding: gzip, chunked
   <dd>See <xref target="gzip.coding"/>.</dd>
 </dl>
 <t>
-   The compression codings do not define any parameters. Their presence
-   &SHOULD; be treated as an error.
+   The compression codings do not define any parameters. The presence
+   of parameters with any of these compression codings &SHOULD; be treated
+   as an error.
```

Comment:
A similar statement appears for the chunked transfer coding, where "their presence" is disambiguated since only the parameters appear as a plural noun, with only the single coding being discussed.